### DISCUSSION TITLE
Configuration Snapshot management

### DISCUSSION THREAD

**Discussion by tocguy** [supermerill/SuperSlicer#3600] [Q&A]
How does one manage the Config Snapshots recorded in SS? I'm running Version: 2.4.58.5 on Win10.

Menu -> Configuration -> Configuration Snapshots... displays the recorded snapshots but the only option is 'Activate'. There no purge/delete/erase or other file management type tool.

Am I missing something?

**Comment by neophyl** [comment]
Nope theres nothing to manage them.  Its a straight copy of the original PS function.  That was something Prusa added.

You could find the folder they are stored in and delete if it bothers you that much.  

It's not something I ever use anyway as I prefer the older method built in from the original Slic3r of exporting a configuration bundle to back up all my custom profiles.  That way I can store them elsewhere away from my pc and easily move them around and restore across multiple machines.

**Comment by ckiefer** [comment]
What needs to be considered / taken care of when updating from one version of SuperSlicher to the next one regarding the profiles for print, printers, and filaments? In my case, I want to update from 2.5.59.3 to 2.5.59.7.

**Comment by tocguy** [comment]
I've found that Superslicer can be treated as a portable app, i.e., the apps files can be replaced/overwritten en masse.
